cumulusci.yml

```diff
@@ -128,22 +128,13 @@ tasks:
                 - record_type: Opportunity.NPSP_Default
                   default: true
 
-    test_data_bdi:
-        description: 'Generate and load test data for BDI benchmarks into an org'
-        class_path: tasks.generate_bdi_data.GenerateBDIData
-        options:
-            mapping_yaml: 'datasets/bdi_benchmark/mapping.yml'
-            num_records: 5000
-            debug_db_path: /tmp/temp_db.db
-
-    performance_tests:
-        description: Runs Robot Framework performance tests
+    test_performance:
+        description: Runs Robot Framework performance tests for BDI Complex Objects
         class_path: cumulusci.tasks.robotframework.Robot
         group: Performance Tests
         options:
             suites: robot/Cumulus/perftests/
-            options:
-                outputdir: robot/Cumulus/results
+            outputdir: robot/Cumulus/results
```

Comment:
This was right before. outputdir is one of the options that gets passed into Robot Framework, not one of the options for the CCI task.

Reply:
Okay.

But if we're going to pick and choose which Robot options to support, we might as well just make them task options.

Reply:
But you probably agree with that and we just haven't gotten around to it.

Reply:
So in the meantime, unless I'm missing something, this still needs to adjusted if you don't want the results to end up in the default outputdir.

Reply:
I was missing something, which is that this snippet is outdated.